### PR TITLE
chore: add avs send http response message

### PR DESCRIPTION
**Pull request by alexandreferris**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [X] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [X] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [X] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

We were sending messages/notifying the other user via proteus/normal request but not also via AVS.

### Solutions

Call to `wcall_resp` on AVS to notify of any calling change.

### CODE REVIEW

File: logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt [hunk 1]
```diff
@@ -198,7 +198,44 @@ actual class CallManagerImpl(
...
+        scope.launch {
+            messageString?.let { message ->
+                withCalling {
+                    when (sendCallingMessage(conversationId, avsSelfUserId, avsSelfClientId, message)) {
+                        is Either.Right -> {
+                            wcall_resp(
+                                inst = deferredHandle.await(),
+                                status = 200,
+                                reason = "",
+                                arg = null
```
Comment (ohassine) [inline logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt:216]: We need to pass something here not null, we get it from sendHandler (first parameter)

File: logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt [hunk 2]
```diff
@@ -198,7 +198,44 @@ actual class CallManagerImpl(
...
+                            )
+                        }
+                        is Either.Left -> {
+                            wcall_resp(
+                                inst = deferredHandle.await(),
+                                status = 400,
+                                reason = "Couldn't send Calling Message",
+                                arg = null
```
Comment (ohassine) [inline logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt:224]: We need to pass the correct error code here, it's not always 400.
and same thing for the arg, a value is needed

Comment (ohassine) [inline logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt:224]: We need to pass the correct error code here, it's not always 400.
and same thing for the arg, a value is needed

**Review by ohassine — CHANGES_REQUESTED** [review]
That's something we have missed 👍 . overall looks good ✅
I've just requested some small changes regarding some required params

File: logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt (hunk 2 above)
Comment (alexandreferris) [reply logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt]: Added a TODO so we can handle the errorCode later from `CoreFailure`

Comment (alexandreferris) [reply logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt]: Added a TODO so we can handle the errorCode later from `CoreFailure`

**Review by ohassine — APPROVED** [review]
lgtm ✅

**Pull request merged**